recipes/pathspec/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pathspec" %}
+{% set version = "0.5.9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 54a5eab895d89f342b52ba2bffe70930ef9f8d96e398cccf530d21fa0516a873
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - pytest
+  imports:
+    - pathspec
+  commands:
+    - pytest --pyargs pathspec  # [unix]
+    # skip symlink tests on windows
+    # https://github.com/cpburnz/python-path-specification/commit/d268375a4c6aa
+    - pytest --pyargs pathspec -k "not test_2_1_check_realpath"  # [win]
```

Comment:
Here's the fail from [appveyor](https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/22388359#L872): I guess I'm not too surprised about symlinks misbehaving there. I see no harm in skipping it with `-k not test_2_1_check_realpath`
```
================================== FAILURES ===================================
____________________ IterTreeTest.test_2_1_check_realpath _____________________
self = <pathspec.tests.test_util.IterTreeTest testMethod=test_2_1_check_realpath>
    def test_2_1_check_realpath(self):
    	"""
    	Tests whether `os.path.realpath` works properly with symlinks.
    	"""
    	# NOTE: Windows does not follow symlinks with `os.path.realpath`
    	# which is what we use to detect recursion. See <https://bugs.python.org/issue9949>
    	# for details.
    	broken_realpath = None
    	try:
    		self.require_symlink()
    		file = os.path.join(self.temp_dir, 'file')
    		link = os.path.join(self.temp_dir, 'link')
    		self.mkfile(file)
    		os.symlink(file, link)
    
    		try:
>   			self.assertEqual(os.path.realpath(file), os.path.realpath(link))
E      AssertionError: 'C:\\Users\\appveyor\\AppData\\Local\\Temp\\1\\tmpras3doid\\file' != 'C:\\Users\\appveyor\\AppData\\Local\\Temp\\1\\tmpras3doid\\link'
E      - C:\Users\appveyor\AppData\Local\Temp\1\tmpras3doid\file
E      ?                                                    ^ ^^
E      + C:\Users\appveyor\AppData\Local\Temp\1\tmpras3doid\link
E      ?                                                    ^ ^^
..\_test_env\lib\site-packages\pathspec\tests\test_util.py:154: AssertionError
=============== 1 failed, 38 passed, 4 skipped in 0.19 seconds ================
(%PREFIX%) %SRC_DIR%>IF 1 NEQ 0 exit /B 1 

```